content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -285,6 +285,17 @@ For Gateway deployments:
    # ...
    ```
 
+### OpenTelemetry Operator for Kubernetes
+
+To use the OpenTelemetry Operator:
+1. See the [official documentation for deploying the OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator#readme). As detailed there, you need to deploy the certificate manager in addition to the Operator.
+
+2. Deploy the OpenTelemetry Collector using one of the standard configurations:
+   * [Daemonset Deployment](#daemonset-deployment)
+   * [Gateway Deployment](#gateway-collector-service)
+
```

Comment:
May be adding  a deploy config with operator will make it easier for users/

Reply:
Done.